microsoft/BotFramework-WebChat
# Add aria-live to New messages

Fixes #2669 

![image](https://user-images.githubusercontent.com/14900841/71035372-2d172780-20d0-11ea-959b-c068cc728435.png)


## Changelog Entry

 

## Description
New messages button itself is not accessible. This adds an aria-label to it

---

-  ~[ ] Testing Added~
   

Manual testing: 
1. Run Web Chat 
1. Type 'help'
1. Scroll up a bit, so bottom of transcript is hidden
1. Open Developer Tools > Elements
1. Confirm that a span above the New messages button exists with the value being 'New messages'
